lib/i18n/tasks/translators/base_translator.rb

```diff
@@ -51,7 +82,7 @@ def fetch_translations(list, opts)
       # @param [Array<[String, Object]>] list of key-value pairs
       # @return [Array<String>] values for translation extracted from list
       def to_values(list, opts)
-        list.map { |l| dump_value(l[1], opts) }.flatten.compact
+        list.map { |l| dump_value(l[1], opts) }.compact
```

Comment:
This could probably go out as a "fix" on it's own - It was compacting array YAML values into the base array which caused the corresponding array size not to match (and also attempted to translate each line as an individual value)

Reply:
Seems like this might actually not be entirely correct.

Our usage is that array values are joined through Slim into paragraphs but obviously there can be cases where an array isn't intended to be used that way. 
We would expect the array to be compacted and translated in its whole context but there's an example in the test files of "encouragement" which doesn't seem to fit this model.

Thoughts on how to proceed?

Reply:
Got it :)